src/SceneManager.ts

```diff
@@ -299,8 +299,20 @@ export class SceneManager {
   }
 
   /**
-   * Subscribe to Gazebo topics required to render a scene. This include
-   * /world/WORLD_NAME/dynamic_pose/info and /world/WORLD_NAME/scene/info
+   * Allows clients to subscribe to a custom topic.
+   *
+   * @param topic The topic to subscribe to.
+   */
+  public subscribeToTopic(topic: Topic): void {
+    this.transport.subscribe(topic);
+  }
```

Comment:
This makes sense to me.  However, I think it would be good to also have a way to unsubscribe, something like:

```
public unsubscribeFromTopic(name: String): void {
  this.transport.unsubscribe(name);
}
```

Reply:
Thanks! Yes, having the unsubscribe exposed is going to be useful as well.

Done in https://github.com/gazebo-web/gzweb/pull/22/commits/af5072b84a641a97038cbc6237fcfe3bf55535df